pom.xml

```diff
@@ -82,7 +82,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.main</groupId>
       <artifactId>jenkins-war</artifactId>
-      <version>2.60.3</version>
+      <version>2.110</version>
```

Comment:
We prefer to use LTS baselines here.

Rather than doing this change, I would suggest just adding the method with

```java
/* TODO uncomment when core dep ≥ 2.110:
@Override
*/
```

since the JVM will still consider it an override when used against sufficiently new cores.

Reply:
it's done.